src/lib/LogDisplayer.svelte

```diff
@@ -20,4 +30,23 @@
     .log-displayer__entry:first-of-type {
         padding-top: 0;
     }
+
+    .log-displayer__entry-group {
+        position: relative;
+    }
+
+    section ul.log-displayer__entry-group + ul.log-displayer__entry-group {
+        border-top: 1px solid var(--main-color-light);
+        padding-top: 20px;
+        margin-top: 20px
+    }
```

Comment:
pourquoi pas reprendre la meme technique que ce qu'on fait sur les log-displayer__entry avec un first-of-type plutot que ce grand selecteur CSS ?

Reply:
j'ai mis un comment pour que ça soit + clair